### PR TITLE
fix: catch not found exception

### PR DESCRIPTION
**Pull request by pnal**
When importing item with not yet known PCI the import should not break cause it is a valid case. The problem was an exception which been not intercepting.

### CODE REVIEW

File: model/portableElement/storage/PortableElementRegistry.php [hunk 1]
```diff
@@ -306,7 +306,11 @@ public function getLatestVersion($identifier)
 
     public function getLatestCompatibleVersion(string $identifier, string $targetVersion): ?PortableElementObject
     {
-        $registered = $this->getAllVersions($identifier);
+        try {
+            $registered = $this->getAllVersions($identifier);
+        } catch (PortableElementNotFoundException $e) {
+            return null;
```
Comment (andreluizmachado) [inline model/portableElement/storage/PortableElementRegistry.php:312]: Can we log it at least?

Comment (andreluizmachado) [reply model/portableElement/storage/PortableElementRegistry.php:312]: Can we have unit tests?

**Review by bartlomiejmarszal — CHANGES_REQUESTED** [review]
Should we log error when catching this exception?

**Review by bartlomiejmarszal — APPROVED** [review]

**Review by andreluizmachado — APPROVED** [review]
- [x] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

**Pull request merged**